js/actions/bookmarkActions.js

```diff
@@ -4,15 +4,18 @@
 
 'use strict'
 
+const Immutable = require('immutable')
 const siteUtil = require('../state/siteUtil')
 const appActions = require('./appActions')
 const windowActions = require('./windowActions')
 const eventUtil = require('../lib/eventUtil')
+const appStoreRenderer = require('../stores/appStoreRenderer')
 const {SWITCH_TO_NEW_TABS} = require('../constants/settings')
 const getSetting = require('../settings').getSetting
 
 const bookmarkActions = {
-  openBookmarksInFolder: function (allBookmarkItems, folderDetail) {
+  openBookmarksInFolder: function (folderDetail) {
```

Comment:
this one should also be converted to an action. Not sure why this is called `bookmarkActions`, probably just some legacy reason because it's not an action the way we define them now